app/pandas.ipynb

```diff
@@ -533,49 +1219,65 @@
     "    return moving_averages[:10]\n",
     "\n",
     "%time unknown_signature()"
-   ],
-   "metadata": {
-    "collapsed": false
-   }
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "### Solution using Pandas and Numpy\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "### Solution using Pandas and Numpy\n"
+   ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
-   "source": [
-    "# write your code here\n"
-   ],
+   "execution_count": 46,
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: total: 453 ms\n",
+      "Wall time: 653 ms\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "[46.8, 52.6, 59.4, 68.0, 79.4, 67.6, 69.8, 67.2, 60.0, 48.8]"
+      ]
+     },
+     "execution_count": 46,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# write your code here\n"
```

Comment:
It seems like the code for calculating the moving average using Pandas and Numpy is missing. You need to implement the solution using Pandas, which should be concise and efficient. Consider using the `rolling` method from Pandas to calculate the moving average.